### ISSUE TITLE
||sailthru.com^ blocks newsletter.businessinsider.com

### ISSUE THREAD

**Issue by Eugene-Savenko** [AdguardTeam/AdGuardSDNSFilter#1032]
> I've been noticing over the last week or so that certain sites are getting blocked by AdGuard even though I have them whitelisted. For example, [newsletter.businessinsider.com](http://newsletter.businessinsider.com/) gets blocked even though I have it in my whitelist. And there are a few others as well. The only way I can get it to load is to disable DNS filtering so that seems to be the issue as far as I can see. Not sure why DNS filtering isn't respecting whitelists?

```
nslookup newsletter.businessinsider.com dns.adguard.com
Server:		dns.adguard.com
Address:	94.140.15.15#53

Non-authoritative answer:
Name:	newsletter.businessinsider.com
Address: 0.0.0.0
```

and

```
nslookup newsletter.businessinsider.com 1.1.1.1
Server:		1.1.1.1
Address:	1.1.1.1#53

Non-authoritative answer:
newsletter.businessinsider.com	canonical name = cb.sailthru.com.
cb.sailthru.com	canonical name = ha-link.sailthru.com.
Name:	ha-link.sailthru.com
Address: 3.226.166.212
```

**Comment by Eugene-Savenko** [comment]
And here we have a report from the [sailthru.com](http://my.sailthru.com/) domain owners. Not filing it as a separate issue, since the problem is related.


> We received a report from one of our users that our platform (a B2B SaaS platform for marketers), is unreachable when AdGuard is enabled. 
> 
> It looks like our [sailthru.com](http://sailthru.com/) domain has been flagged. Is it possible to exclude our [sailthru.com](http://sailthru.com/) domain from the flag? While our domain is included in emails that we send for reporting purposes, we don't serve up any ads nor share cookie information with any ad providers. See our [privacy policy here](https://www.sailthru.com/legal/privacy-statement/).
> 
> At the very least, could an exclusion for [my.sailthru.com](http://my.sailthru.com/) be included?
